### DISCUSSION TITLE
Are the cuda dll's needed? (For using cuda enabled whisper not building)

### DISCUSSION THREAD

**Discussion by gartia** [ggml-org/whisper.cpp#2204] [Q&A] [answered]
I am using the c++ api so runtime loading whisper.dll and i'm not including the cudart or cublas dll's in the folder or specifying them at all but i am still getting a "whisper_backend_init: using CUDA backend" 
I'm curious if my system might have them loaded from some other application or they just aren't required for use of whisper.dll?

**Comment by ulatekh** [comment, marked as answer]
Sounds like CUDA is installed on your machine. You mentioned DLLs, so I assume you're running MS Windows. Check your "Apps & features" settings for any package with CUDA in the name, e.g. "NVIDIA CUDA Toolkit".

**Comment by gartia** [comment]
Looks like i did infact have a previous installation that was semi installed for some reason but the dll directory was intact! Normally use cuda toolkit on wsl so it was probably old. My bad!
